src/ansys/dpf/composites/layup_info/_add_layup_info_to_mesh.py

```diff
@@ -71,21 +76,28 @@ def add_layup_info_to_mesh(
         dictionary key in the :attr:`.ContinuousFiberCompositesFiles.composite`
         attribute. This parameter is only required for assemblies.
         See the note about assemblies in the description for the :class:`.CompositeModel` class.
+    rst_stream_provider:
```

Comment:
In most libraries, there is a space before the colon and then a space and the data type. Is yours a bit different for a reason?

Reply:
The type is visible in the function header directly. I think it is not needed here again.